cmd/restic/cmd_backup.go

```diff
@@ -103,7 +103,7 @@ func init() {
 	cmdRoot.AddCommand(cmdBackup)
 
 	f := cmdBackup.Flags()
-	f.StringVar(&backupOptions.Parent, "parent", "", "use this parent `snapshot` (default: last snapshot in the repo that has the same target files/directories)")
+	f.StringArrayVar(&backupOptions.Parents, "parent", nil, "use this parent `snapshot` (can be specified multiple times, default: last snapshots in the repo that include the same target)")
```

Comment:
is the default multiple snapshots now?  It looks like the default is still a single snapshot.

Reply:
The first commits of this PR are from #3121 which adds support for multiple parent snapshots. You are right, I just chose the commits which only allow to manually specify multiple parents but not the commit which searcher for multiple fitting snapshots. So this is misleading.

However, anyway the order is that #3121 (or at least parts of it) should be merged first and then I'll rebase this PR such that it only contains the last three commits. So there will be no change in this line by this PR.